settings.gradle.kts

```diff
@@ -43,8 +43,9 @@ includeSharedLib("agent-instrumentation")
 includeSharedLib("jvmapi")
 includeSharedLib("knasm")
 includeSharedLib("konform")
-includeSharedLib("interceptor-hook")
-includeSharedLib("interceptor-http")
+// FYI: Interceptor not patched for macOS ARM64 architecture.
+//includeSharedLib("interceptor-hook")

```

Comment:
haven't we had it excluded from the build by default anyway?

Reply:
Have to ask @epamrsa.
Comment it out because the build fails with these modules.

Reply:
It will totally disable build with drill-hook. Wrong solution :)

Reply:
as it was discussed: interceptor-hook and interceptor-http can be commented